SanteDB.Persistence.Data.ADO/Services/Persistence/ActPersistenceService.cs

```diff
@@ -79,7 +79,7 @@ public override Core.Model.Acts.Act ToModelInstance(object dataInstance, DataCon
             Act retVal = null;
 
             // 
-            switch (dbAct.ClassConceptKey.ToString().ToUpper())
+            switch (dbAct.ClassConceptKey.ToString().ToLower())
```

Comment:
The actual code file this is .ToLowerInvariant, for some reason GH is showing it as .ToLower?